package.json

```
@@ -1,7 +1,7 @@
 {
     "name": "@hapi/tlds",
     "description": "TLDS list for domain validation",
-    "version": "1.0.2",
+    "version": "1.0.3",
```

Never bump version when submitting a PR
```suggestion
    "version": "1.0.2",
```

---

Oh yeah right, I missed it.